batchflow/models/torch/callbacks/plateau.py

```diff
@@ -0,0 +1,86 @@
+""" Callbacks to perform on loss plateau. """
+import numpy as np
+
+from .base import BaseCallback
+
+
+
+class PlateauCallback(BaseCallback):
```

Comment:
if this class dont have an action to do. what about rename it to BasePlateauCallback class?